README.md

```diff
@@ -1,5 +1,16 @@
 # Helium GraphQL Documentation
 
+## Documentation Standards
+- For queries and mutations
+    - For every query and mutation, there needs to be a clear, easy to understand explanation of what the query or mutation does
+    - For each argument of each query and mutations there needs to be a description of what the argument represents and where it can be found in our existing platform
+    - For argument names regarding Ids, if the arguments represents the id of course, name the argument courseId instead of just course or id 
```

Comment:
This is more related to the actual coding of GraphQL operations more than it is documentation, so not sure this is the best place for it.

Reply:
The first two points relate strictly to docs but I can remove the 3rd